model/jpa/src/main/resources/META-INF/jpa-changelog-authz-4.0.0.CR1.xml

```diff
@@ -74,5 +74,21 @@
                 <constraints nullable="false"/>
             </column>
         </addColumn>
+
+        <createTable tableName="RESOURCE_ATTRIBUTE">
+            <column name="ID" type="VARCHAR(36)" defaultValue="sybase-needs-something-here">
+                <constraints nullable="false"/>
+            </column>
+            <column name="NAME" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="VALUE" type="VARCHAR(255)"/>
+            <column name="RESOURCE_ID" type="VARCHAR(36)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addPrimaryKey columnNames="ID" constraintName="CONSTRAINT_RESOURCE_ATTRIBUTE_PK" tableName="RESOURCE_ATTRIBUTE"/>
```

Comment:
The CONSTRAINT_RESOURCE_ATTRIBUTE_PK identifier is too long for Oracle databases, 30 characters is max value